File: src/main/java/sawyern/cookiebot/models/entity/Cookie.java
package sawyern.cookiebot.models.entity;


import lombok.*;

import javax.persistence.*;

@Entity
@Table(name = "COOKIE")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@EqualsAndHashCode(callSuper = true)
public class Cookie extends DbItem {
    @ManyToOne
    @JoinColumn(name = "ACCOUNT_ID")
    private Account account;

    @Column(name = "type")
    public String type;
}

File: src/main/kotlin/sawyern/cookiebot/models/Cookie.kt
package sawyern.cookiebot.models

import javax.persistence.*

@Entity
@Table(name = "COOKIES")
data class Cookie(
    @ManyToOne
    @JoinColumn(name = "ACCOUNT_ID")
    var account: Account,

    @ManyToOne
    @JoinColumn(name = "SEASON_ID")
    var season: Season
): DbItem()